cmd/root.go

```diff
@@ -164,11 +185,22 @@ func ensureToken(ctx context.Context, requiredScopes []string) (context.Context,
 				return ctx, fmt.Errorf("error authenticating the API token: %w", err)
 			}
 			log.WithContext(ctx).Debug("successfully authenticated")
-			apiKey = resp.Msg.GetAccessToken()
+			accessToken = resp.Msg.GetAccessToken()
 		} else {
 			return ctx, errors.New("OVM_API_KEY does not match pattern 'ovm_api_*'")
 		}
-		return context.WithValue(ctx, sdp.UserTokenContextKey{}, apiKey), nil
+
+		// Validate that the API_KEY has the required scopes
```

Comment:
There's already https://github.com/overmindtech/cli/blob/e9a7d3a2eefa1d0562ae86874bc379fbdb4cb461/cmd/root.go#L235-L239 below to check that?